### DISCUSSION TITLE
How to boot from SRP target?

### DISCUSSION THREAD

**Discussion by gobiursus** [ipxe/ipxe#1220] [General]
How do I boot from Infiniband SRP target? The documents only shows Fiber Channel and iSCSI in `sanurl` or `sanboot`.

**Comment by mcb30** [comment]
I've updated https://ipxe.org/sanuri to include the IB SRP URI format.

**Reply by gobiursus** [reply]
Thank you. Is this also available for `sanhook`, `sanboot` or `sanunhook`?

**Reply by mcb30** [reply]
Yes, the SAN URI format is the same wherever it is used within iPXE.

**Reply by gobiursus** [reply]
Thank you

**Reply by gobiursus** [reply]
Hi,
I successfully hooked it but I still can't boot. No information in `dmesg` on my SAN(IB SRP SCST). But I will create another issue. Thank you again
